packages/core/src/App/Containers/AccountSignupModal/account-signup-modal.jsx

```diff
@@ -249,32 +254,8 @@ const AccountSignupModal = ({
             />
         </Dialog>
     );
-};
+});
+
 
-AccountSignupModal.propTypes = {
-    clients_country: PropTypes.string,
-    disableApp: PropTypes.func,
-    enableApp: PropTypes.func,
-    is_loading: PropTypes.bool,
-    is_logged_in: PropTypes.bool,
-    is_mobile: PropTypes.bool,
-    is_visible: PropTypes.bool,
-    logout: PropTypes.func,
-    onSignup: PropTypes.func,
-    residence_list: PropTypes.arrayOf(PropTypes.object),
-    toggleAccountSignupModal: PropTypes.func,
-};
 
-export default connect(({ ui, client }) => ({
-    is_visible: ui.is_account_signup_modal_visible,
-    toggleAccountSignupModal: ui.toggleAccountSignupModal,
-    enableApp: ui.enableApp,
-    disableApp: ui.disableApp,
-    is_loading: ui.is_loading,
-    is_mobile: ui.is_mobile,
-    onSignup: client.onSignup,
-    is_logged_in: client.is_logged_in,
-    residence_list: client.residence_list,
-    clients_country: client.clients_country,
-    logout: client.logout,
-}))(AccountSignupModal);
+export default (AccountSignupModal);
```

Comment:
```suggestion
export default AccountSignupModal;
```
This exists in more places @yaswanth-deriv